tests/data/test_docs.py

```diff
@@ -328,3 +328,67 @@ def test_delete_scalardocs(self, test_mp, request):
 
         with pytest.raises(Docs.NotFound):
             test_mp.backend.optimization.scalars.docs.get(scalar.id)
+
+
+@database_platforms
+class TestDatabaseDocs:
```

Comment:
Why is this only tested for databases?

Reply:
Thanks, forgot to remove that after I added the other layers.